java/apps/authentication-portal/src/main/webapp/fido2-auth.jsp

```diff
@@ -174,11 +178,17 @@
     <script type="text/javascript" src="libs/base64js/base64js-1.3.0.min.js"></script>
     <script type="text/javascript" src="libs/base64url.js"></script>
 
+    <%
+        String myaccountUrl = application.getInitParameter("MyAccountURL");
+        if (StringUtils.isEmpty(myaccountUrl)) {
+            myaccountUrl = ServiceURLBuilder.create().addPath(MY_ACCOUNT).build().getAbsolutePublicURL();
+        }
+    %>
     <script type="text/javascript">
         $(document).ready(function () {
-            var myaccountUrl = '<%=application.getInitParameter("MyAccountURL")%>';
+            var myaccountUrl = '<%=myaccountUrl%>';
 
-            if ("<%=tenantDomain%>" !== "" || "<%=tenantDomain%>" !== "null") {
+            if ("<%=tenantDomain%>" !== "" && "<%=tenantDomain%>" !== "null") {
```

Comment:
Shall we also encode tenantDomain param?

Ref: https://github.com/wso2-enterprise/wso2-apim-internal/issues/3391 and https://github.com/wso2/identity-apps/pull/3945

Reply:
+1.
Addressed with https://github.com/wso2/identity-apps/pull/3966/commits/8f648df8db4a18523de502d843b4d2bb2839f2dc